sympy/matrices/determinant.py

```diff
@@ -569,6 +571,10 @@ def _det(M, method="bareiss", iszerofunc=None):
 
         If it is set to ``'berkowitz'``, Berkowitz' algorithm will be used.
 
+        If it is set to ``'bird'``, Bird' algorithm will be used.
+
+        If it is set to ``'laplace'``, Laplace' algorithm will be used.
```

Comment:
```suggestion
        If it is set to ``'laplace'``, Laplace's algorithm will be used.
```

Reply:
Yes, thanks for spotting. Addressed in a777540c78ce8da5ca9e68d4182f68b6d299a231